cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/CommonGridModelExportTest.java

```diff
@@ -470,6 +470,7 @@ void testFaraoUseCaseManualExport() throws IOException {
             String country = n.getSubstations().iterator().next().getCountry().orElseThrow().toString();
             CgmesMetadataModel sshModel = n.getExtension(CgmesMetadataModels.class).getModelForSubset(CgmesSubset.STEADY_STATE_HYPOTHESIS).orElseThrow();
             sshModel.clearDependencies()
+                    .clearSupersedes()
```

Comment:
The test has no assertion linked to our changes.

Reply:
In the line 510 below we check that the exported XML file contains exactly only one "supersede", that is the one that has been added by the user. The model used in the test contains a previous "supersede", so we are checking that the previous has been removed.